### DISCUSSION TITLE
Is possible ignore the changes in the line of OpenAPI spec version ?

### DISCUSSION THREAD

**Discussion by MedinaGitHub** [orval-labs/orval#1213] [Q&A] [answered]
When updating a single value in our model, I notice that hundreds of files are marked as modified, the only modification being the version of the OpenAPI specification. This situation is not only annoying but also complicates code reviews.  Is there any way to pass parameters to specifically ignore changes to that line...?

Something like: 
 "output": {
   _"ignoreSwaggerVersion": true,_
picture of my problem
<img width="1317" alt="image" src="https://github.com/anymaniax/orval/assets/16387514/45524ab3-7dde-4390-8677-c092fcc29e72">

**Comment by MedinaGitHub** [comment]
@soartec-lab  Do you think that is possible? Thanks !

**Reply by melloware** [reply]
I would think with a configurtion option anything is possible.

**Reply by soartec-lab** [reply]
@MedinaGitHub 
Good, if I were you I would name it `ignoreOpenApiSpecVersion`.

**Reply by MedinaGitHub** [reply]
Ok, I'll work on it and then I will make the PR. Thanks, guys!

**Reply by MedinaGitHub** [reply, marked as answer]
guys, I found the "header" method and it works for my needs. thanks! 


```
  petstore: {
    output: {
      override: {
        header: (info: InfoObject): String[] => [
          `Generated by orval 🍺`,
          `Do not edit manually.`,
          ...(info.title ? [info.title] : []),
          ...(info.description ? [info.description] : []),
          ...(info.version ? [`OpenAPI spec version: ${info.version}`] : []),
        ],
      },
    },
```
  },

**Reply by strazdinsg** [reply]
Thanks, this saved some headache for me. I got annoyed that every time my backend API version changes, absolutely all the generated endpoint and module files get a Git change, because the version field in the comment is changed in every single file. :)
